.php_cs.dist

```diff
@@ -2,16 +2,17 @@
 
 $finder = PhpCsFixer\Finder::create()->in([__DIR__ . '/src', __DIR__ . '/tests']);
 
-return PhpCsFixer\Config::create()
+return (new PhpCsFixer\Config)
     ->setRules([
         '@PSR2' => true,
         'array_syntax' => ['syntax' => 'short'],
         'binary_operator_spaces' => true,
-        'blank_line_before_return' => true,
+        'blank_line_before_statement' => true,
```

Comment:
`composer run php-cs-fixer:lint` marked `blank_line_before_return` as deprecated and should replaced with `blank_line_before_statement`